pkg/cmd/experimental/syncgroups/util/util.go

```diff
@@ -0,0 +1,122 @@
+package config
```

Comment:
util packages are usually a dumping ground.  Do you have a better name?  Oh, and make your package name match the folder name.